src/main/java/gross/citibike/map/MapController.java

```diff
@@ -0,0 +1,87 @@
+package gross.citibike.map;
+
+import gross.citibike.lambda.LambdaService;
+import gross.citibike.lambda.LambdaServiceFactory;
+import gross.citibike.lambda.Request;
+import gross.citibike.lambda.Response;
+import gross.citibike.service.StationResponse;
+import io.reactivex.rxjava3.core.Single;
+import org.jxmapviewer.viewer.GeoPosition;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.lang.annotation.Repeatable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jxmapviewer.viewer.Waypoint;
+
+public class MapController {
+    private MapFrame frame;
+    private int maxPoints = 2;
+
+    public MapController() {
+        frame = new MapFrame();
+        setupListeners();
+        frame.setVisible(true);
+    }
+
+    private void setupListeners() {
+        frame.getMapComponent().addMouseListener(new java.awt.event.MouseAdapter() {
+            @Override
+            public void mouseClicked(java.awt.event.MouseEvent e) {
+                GeoPosition position = frame.getMapComponent()
+                        .convertPointToGeoPosition(e.getPoint());
+                addPoint(position);
+            }
+        });
+
+        frame.getMapButton().addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (frame.getMapComponent().getWaypoints().size() == 2) {
+                    LambdaService lambdaService = LambdaServiceFactory.getService();
```

Comment:
You are calling it like it's `static` by using the capital L.
You need to **create** a LambdaServiceFactory and then call getService() on it.